org-roam-db.el

```diff
@@ -224,6 +228,13 @@ This is equivalent to removing the node from the graph."
     :values $v1]
    (list (vector file titles))))
 
+(defun org-roam-db--insert-headlines (headlines)
+  "Insert TITLES for a FILE into the Org-roam cache."
```

Comment:
Just to be clear, a lot of the code is still WIP and not ready for review.  Things like this will typically get ironed out later in the process.  Right now, it's only about getting a working prototype.

Reply:
To answer your question, though:
https://github.com/org-roam/org-roam/blob/66842f409ee5cbf0028a1686755abe179c12ef38/org-roam-db.el#L217-L222

This is explained in the docstring for `org-roam--extract-links`:
```
Extracts all link items within the current buffer.
Link items are of the form:

    [from to type properties]

This is the format that emacsql expects when inserting into the database.
FILE-FROM is typically the buffer file path, but this may not exist, for example
in temp buffers.  In cases where this occurs, we do know the file path, and pass
it as FILE-PATH.
```

So, since `extract-links` already preformats the vector, `insert` doesn't need further processing.

Reply:
Addressed the docstring in 17f58e3122277454fa74250aa3046328060f654a.